crates/uv/tests/it/build_backend.rs

```diff
@@ -301,7 +301,7 @@ fn rename_module() -> Result<()> {
         module-name = "bar"
 
         [build-system]
-        requires = ["uv_build>=0.5,<0.7"]
+        requires = ["uv_build>=0.5,<0.8"]
```

Comment:
```suggestion
        requires = ["uv_build>=0.7,<0.8"]
```

Reply:
We could also do `<10000` here, the no-upper-bound warning is silenced as long as there is any upper bound at all-

Reply:
Does this matter?

As an aside, updating these manually is really a bummer. It'd be great to template these in the test suite based on the current version.

Reply:
Ah that's why there's a bound? I see